objc/core/A0LockReact.m

```diff
@@ -44,6 +49,19 @@ - (void)configureLockWithClientId:(NSString *)clientId domain:(NSString *)domain
     _lock = [A0Lock newLockWithClientId:clientId domain:domain];
 }
 
+- (void)hideWithCallback:(A0LockCallback)callback {
+    if (self.shown) {
```

Comment:
Thanks! FWIW one other issue this has allowed us to work around.

During development of a react-native app, if the lock is open and you hit 'command r' to refresh the app, the lock will persist open (since it lives in the native layer) even though the js app state has been blown away and restarted. We're able to work around this now by calling lock.hide() on init of app to ensure the lock is closed.